modular_skyrat/modules/customization/modules/language/piratespeak.dm

```diff
@@ -1,6 +1,6 @@
 /datum/language/piratespeak
-	name = "Pirate Gutter"
-	desc = "The language of space pirates. Sounds similar to gutter, though intentionally filled with odd slang to make deciphering hard."
+	name = "Spatial Sea-talk"
```

Comment:
```suggestion
	name = "Spatial Sea-tongue"
```
Sea-talk sounds weird in my personal opinion; tongue may not be better tho so its up for debate.
Also worth considering, "Spacial" as a synonym which may fit better to emphasize solar-space vs material-space (https://thesaurus.plus/related/spacial/spatial)